cli/cmd/install.go

```diff
@@ -61,6 +61,10 @@ func installPlugin(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get team name: %w", err)
 	}
+
+	pluginVersionWarner, _ := managedplugin.NewPluginVersionWarner(log.Logger, authToken.Value)
+	specs.WarnOnOutdatedVersions(ctx, pluginVersionWarner, sources, destinations, transformers)
```

Comment:
This will panic on error. Either add a `nil` check here, or a nil check in the method.

Reply:
Are you sure?
https://github.com/cloudquery/plugin-pb-go/blob/main/managedplugin/version_checker.go#L58

Reply:
Added the check. But it cannot fail at the moment (hubClient always succeeds). It's still good practice to keep the check.